app/src/main/java/com/example/lastfm/Post.java

```diff
@@ -0,0 +1,39 @@
+package com.example.lastfm;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Post {
```

Comment:
The name of the class doesn't match its content. Rename it.

Reply:
Deleted.